commons-numbers-combinatorics/src/test/java/org/apache/commons/numbers/combinatorics/FactorialDoubleTest.java

```diff
@@ -121,10 +124,31 @@ public void testCacheDecrease() {
      * Direct multiplication implementation.
      */
     private double factorialDirect(int n) {
+
         double result = 1;
+
         for (int i = 2; i <= n; i++) {
             result *= i;
         }
+
         return result;
+
+
     }
+
+  @Test
+  public void testWithCacheThrowsCombinatoricsException() {
+
```

Comment:
Don't keep empty lines. by guidelines after method name no empty line should be there neither before ending method.